cmd/backup-manager/app/util/remote.go

```diff
@@ -228,10 +228,13 @@ func newGcsStorageOption(conf *gcsConfig) []string {
 func makeS3Config(s3 *v1alpha1.S3StorageProvider, fakeRegion bool) *s3Config {
 	conf := s3Config{}
 
-	conf.bucket = s3.Bucket
+	path := strings.Trim(s3.Bucket, "/") + "/" + strings.Trim(s3.Prefix, "/")
```

Comment:
can we use `path.Join` here?

Reply:
`path.Join` will parse `../`, I don't know if it's safe, so I concatenate string.

Reply:
I tried this on S3 and `..` is a valid name.